apps/editing-toolkit/editing-toolkit-plugin/error-reporting/index.js

```diff
@@ -3,46 +3,80 @@
  */
 import apiFetch from '@wordpress/api-fetch';
 
-/**
- * Errors that happened before this script had a chance to load
- * are captured in a global array. See `./index.php`.
- */
-const headErrors = window._jsErr || [];
-const headErrorHandler = window._headJsErrorHandler;
-
-const reportError = ( { error } ) => {
-	// Sanitized error event objects do not include a nested error attribute. In
-	// that case, we return early to prevent a needless TypeError when defining
-	// `data`, below. Also, sanitized errors don't include any useful information,
-	// so the sensible thing to do is to completely ignore them.
-	if ( ! error ) {
-		return;
-	}
-
-	const data = {
-		message: error.message,
-		trace: error.stack,
-		url: document.location.href,
-		feature: 'wp-admin',
+import * as Sentry from '@sentry/browser';
+import { Integrations } from '@sentry/tracing';
+
+const dataFromPHP = window.dataFromPHP;
+const shouldActivateSentry =
```

Comment:
I think this can be simplified to `window.dataFromPHP?.shouldActivateSentry === 'true'`